File: 백준/Bronze/10818. 최소， 최대/최소， 최대.c
#include <stdio.h>

int main() {
    int i, N;
	int min = 1000000, max = -1000000;
	
    scanf("%d", &N);
    int array[N];
	
    for (i = 0; i < N; i++) {
        scanf("%d", &array[i]);
        if (min > array[i]) {
            min = array[i];
        }
        if (max < array[i]) {
            max = array[i];
        }
    }
	
    printf("%d %d", min, max);
    return 0;
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int min = 1000000, max = -1000000;
        int N = scanner.nextInt();
        int[] array = new int[N];
		
        for (int i = 0; i < N; i++) {
            array[i] = scanner.nextInt();
            
			if (min > array[i]) { 
				min = array[i]; 
			} 
			if (max < array[i]) { 
			max = array[i]; 
			}
        }
		
        System.out.println(min + " " + max);
        scanner.close();
    }
}